scripts-local/tests/ica_redeem.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+echo "Register zone"
+# setup logic on controller zone
+$STRIDE1_EXEC tx stakeibc register-host-zone \
+    connection-0 uatom ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9 channel-0 --chain-id \
```

Comment:
is this the right IBC hash? i thought it was `ibc/27...`

Reply:
This is an old file, going to remove it